discovery_engine_api_tests/tests/back_office/test_actions_with_documents.py

```diff
@@ -0,0 +1,43 @@
+import allure
+import pytest
+
+from handlers.api_handler import ApiHandler
+from model.properties.properties import Properties
+from model.properties.property import Property
+from utils import assert_utils as au
+from utils import test_utils as tu
+
+
+@allure.suite("Test Actions With Documents Test Suite")
+class TestActionsWithDocuments:
+    api_handler = ApiHandler()
+
+    @allure.severity(allure.severity_level.NORMAL)
+    def test_delete_doc(self, ingest_generated_document):
+        request = self.api_handler.delete_document(doc_id=ingest_generated_document)
+        au.assert_status_code_equals(request.status_code, 204)
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: create ticket")
```

Comment:
🔧 ticket already exists
```suggestion
    @pytest.mark.skip(reason="ET-4004")
```